core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -100,7 +100,7 @@ def start_board(self, boards: List[Tuple[FlightControllerType, str]]) -> bool:
             return False
 
         if len(boards) > 1:
-            print(f"More than a single board detected: {boards}")
+            logger.warning(f"More than a single board detected: {boards}")
```

Comment:
does it say which one is going to be used?

Reply:
No. Can add that.

Reply:
Done.